### PR TITLE
WIP: Update mentor guide

### PR DESCRIPTION
**Pull request by klappradla**
Hey @alicetragedy,

I started updating the project submission section in the mentors' guide according the their feedback and the new masterplan for handling project submissions. It's still work in progress and I can also remove it again if there's already a different version in the making.

I tried to keep things short *(e.g. skipped stuff like the CoC field since it's obvious)* - but it's hard 🙈

Feel free to request any changes!

### CODE REVIEW

**Comment by hola-soy-milk** [comment]
Hey @klappradla,

Thanks for creating this and for the wording, I like it a lot!

I have some in-progress wording additions as well, and was wondering if I could cherry-pick them onto here?

**Comment by alicetragedy** [comment]
@klappradla @ramonh thank you both for your work! If that's ok with both of you, I'd like to review the PR once Ramón has added his own changes/suggestions to it. please feel free to ping me once that's been added, could review later today or tomorrow. ✨

**Comment by alicetragedy** [comment]
Update: just looked over the guide and it looks really good to me, I like the wording as well. Maybe we can merge once @ramonh has added his changes :)

**Comment by hola-soy-milk** [comment]
Ok cool, thank you for confirming! I'll go ahead and merge these on, then

File: guide/projects.md [hunk 1]
```diff
@@ -92,6 +94,8 @@ By submitting your project for Rails Girls Summer of Code, you are allowing your
 
 On the “All Projects” page, you can view all submitted projects and their status (proposed, accepted, rejected) as well as edit or delete your submission.
 
+After submitting your project, it's likely that one of the organizers will offer feedback or ask questions. It's a good idea to keep an eye out for these!
```
Comment (alicetragedy) [inline guide/projects.md:97]: 👍

File: guide/projects.md [hunk 2]
```diff
@@ -41,6 +41,8 @@ Having a Code of Conduct or similar guidelines on your project is encouraged. Pr
 Mentors are patient, tolerant and open people! Ideally you are already familiar with the [Rails Girls](http://www.railsgirls.com) movement or other newcomer initiatives. You care about diversity and making Open Source projects more accessible to newcomers.
 A mentor is a maintainer of the proposed project (or a core-contributor) and is the dedicated contact person for the team over the summer.
 
+RGSoC teams have their nuances, but in general a good mentor is one that is approachable and stays informed on the students' progress. After getting acquainted, teams set up their communication channel (oftentimes the RGSoC Slack group) and keep it there. Some student teams will reach out to you often, others will do so only when truly stuck. It's important to get a feel for when it's time to check in with the students and see if there's a way you can help.
```
Comment (alicetragedy) [inline guide/projects.md]: As we will be doing a separate onboarding document for the mentors (things they need to set up, etc...), I think we could remove the following line without losing too much information: "After getting acquainted, teams set up their communication channel (oftentimes the RGSoC Slack group) and keep it there."

**Comment by alicetragedy** [comment]
@ramonh great additions! I made a small comment regarding a sentence which I think could easily be removed, could you do that? I think we're ready to merge after that — thank you!

**Comment by hola-soy-milk** [comment]
Thanks @alicetragedy! I've done as suggested.

**Comment by alicetragedy** [comment]
Thanks @klappradla && @ramonh for this! 👯

**Pull request merged**